pybind11_mkdoc/mkdoc_lib.py

```diff
@@ -306,15 +306,23 @@ def folder_version(d):
             if '-stdlib=libc++' in args:
                 parameters.extend(['-isystem', os.path.join(llvm_dir, 'include', 'c++', 'v1')])
 
-            clang_include_dir = max(
-                glob(os.path.join(llvm_dir, 'lib', 'clang', '*')
-            ), default=None, key=folder_version)
+            if 'CLANG_INCLUDE_DIR' in os.environ:
+                clang_include_dir = os.environ['CLANG_INCLUDE_DIR']
```

Comment:
This could be written with `os.environ.get(...)`, but I think this is fine, possibly more readable.